example/glue/ObjectModel.hpp

```diff
@@ -22,14 +22,39 @@
 #include "ModronAssertions.h"
 #include "modronbase.h"
 #include "objectdescription.h"
+#include "omrgcconsts.h"
 
+#include "ForwardedHeader.hpp"
 #include "HeapLinkedFreeHeader.hpp"
 
 class MM_GCExtensionsBase;
 
 #define J9_GC_OBJECT_ALIGNMENT_IN_BYTES 0x8
 #define J9_GC_MINIMUM_OBJECT_SIZE 0x10
 
+/*
+ * Define type used to represent an object's header (24/56 bits size; 4 bits age, 4 bits flags)
+ */
+#define SCAVENGER_FORWARDING_SLOT_OFFSET		0 /* fomrobject_t offset from object header address to fomrobject_t-sized slot to hold forwarding ptr */
```

Comment:
All of the code specific to the scavenger feature should be under the appropriate build flag